novawallet/Modules/Swaps/Slippage/SwapSlippagePresenter.swift

```diff
@@ -0,0 +1,182 @@
+import Foundation
+import SoraFoundation
+import BigInt
+
+final class SwapSlippagePresenter {
+    weak var view: SwapSlippageViewProtocol?
+    let wireframe: SwapSlippageWireframeProtocol
+    let numberFormatterLocalizable: LocalizableResource<NumberFormatter>
+    let percentFormatterLocalizable: LocalizableResource<NumberFormatter>
+    let completionHandler: (BigRational) -> Void
+    let prefilledPercents: [Decimal] = [0.1, 1, 3]
+    let initPercent: BigRational?
+    let chainAsset: ChainAsset
+    let amountRestriction: (lower: Decimal, upper: Decimal) = (lower: 0.01, upper: 50)
```

Comment:
let's change 0.01 -> 0.1 since we can't input 0.01 based on the view model restrictions.